streamer/src/nonblocking/quic.rs

```diff
@@ -381,7 +381,11 @@ fn handle_chunk(
                     stats.total_invalid_chunks.fetch_add(1, Ordering::Relaxed);
                     return true;
                 }
-                if chunk.offset + chunk_len > PACKET_DATA_SIZE as u64 {
+                let end_of_chunk = match chunk.offset.checked_add(chunk_len) {
+                    Some(end) => end,
+                    None => return false,
```

Comment:
Should we return true, so that the stream will be dropped? If we had received a corrupted chunk, and we drop it, the stream is corrupted anyway..

Reply:
yea